assets/queries/cloudFormation/filter_and_alarm_exist_for_route_table_changes/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_Route_Table_Changes",
+  "queryName": "Filter And Alarm Not Exists For Route Table Changes",
```

Comment:
Change **directory name** and **id** to: `route_table_changes_without_alarms`
Change **query name** to: `Route Table Changes Without Alarms`